ignite/templates/app/files/cmd/{{binaryNamePrefix}}d/cmd/root.go.plush

```diff
@@ -2,67 +2,67 @@ package cmd
 
 import (
 	"os"
-	"strings"
 
+	"github.com/spf13/cobra"
+
+	authv1 "cosmossdk.io/api/cosmos/auth/module/v1"
+	stakingv1 "cosmossdk.io/api/cosmos/staking/module/v1"
 	"cosmossdk.io/client/v2/autocli"
+	"cosmossdk.io/core/address"
+	"cosmossdk.io/core/registry"
 	"cosmossdk.io/depinject"
 	"cosmossdk.io/log"
+	"github.com/cosmos/cosmos-sdk/x/auth/tx"
+	authtxconfig "github.com/cosmos/cosmos-sdk/x/auth/tx/config"
+	"github.com/cosmos/cosmos-sdk/x/auth/types"
+
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/config"
-	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/codec"
 	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
 	"github.com/cosmos/cosmos-sdk/server"
 	"github.com/cosmos/cosmos-sdk/types/module"
-	"github.com/cosmos/cosmos-sdk/x/auth/tx"
-	authtxconfig "github.com/cosmos/cosmos-sdk/x/auth/tx/config"
-	"github.com/cosmos/cosmos-sdk/x/auth/types"
-	"github.com/spf13/cobra"
-	"github.com/spf13/pflag"
-
 	"<%= ModulePath %>/app"
 )
 
 // NewRootCmd creates a new root command for <%= BinaryNamePrefix %>d. It is called once in the main function.
 func NewRootCmd() *cobra.Command {
```

Comment:
Wdyt to return the cmd and an error here, which we can handle with the proper exit status at the high chain level?

Reply:
Personally I like panics as it gives you the stacktrace for free. However, if you really want error handling, I can add 👍

Reply:
IMHO, the panics terminate the CLI program, creating problems when resources need to be correctly closed. I think it makes sense to use panic if we have a good recovery system